samples/node/pub_sub_electron_node/README.md

```diff
@@ -62,64 +60,105 @@ Replace with the following with the data from your AWS account:
 * `<region>`: The AWS IoT Core region where you created your AWS IoT Core thing you wish to use with this sample. For example `us-east-1`.
 * `<account>`: Your AWS IoT Core account ID. This is the set of numbers in the top right next to your AWS account name when using the AWS IoT Core website.
 
-Note that in a real application, you may want to avoid the use of wildcards in your ClientID or use them selectively. Please follow best practices when working with AWS on production applications using the SDK. Also, for the purposes of this sample, please make sure your policy allows a client ID of `test-*` to connect or use `--client_id <client ID here>` to send the client ID your policy supports.
+Note that in a real application, you may want to avoid the use of wildcards in your ClientID or use them selectively. Please follow best practices when working with AWS on production applications using the SDK.
 
 </details>
 
 ## How to run
-
 ### Direct MQTT via mTLS
 
 To Run this sample using a direct MQTT5 connection with a key and certificate, go to the `node/pub_sub_electron_node` folder.
 1. Setup your credential. You need to fill in the credentials in the `node/pub_sub_electron_node/settings.ts` with your AWS endpoint, certificate file path, private key file path.
 
 2. Install node packages
 ``` sh
-npm install
+npm install .
 ```
 
-3. Build and Run
+3. Start Sample
 ```sh
-npm run build
 npm run start
 ```
 
-### Websockets
+### MQTT over Websockets with TLS
 
 To Run this sample using Websockets, go to the `node/pub_sub_electron_node` folder.
 1. Setup your credential. You will need to set your AWS credentials in your environment variables or local files. See the [authorizing direct AWS](https://docs.aws.amazon.com/iot/latest/developerguide/authorizing-direct-aws.html) page for documentation on how to get the AWS credentials, which then you can set to the `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_SESSION_TOKEN` environment variables.
```

Comment:
Trivial: remove "Setup your credential." and replace with "Setup your AWS credentials in your environment variables or local files."?